BaseClasses.py

```diff
@@ -1188,9 +1188,15 @@ def create_playthrough(self, create_paths: bool = True) -> None:
                                          chain.from_iterable(multiworld.precollected_items.values())
                                          if item.advancement])}
 
-        for i, sphere in enumerate(collection_spheres):
-            self.playthrough[str(i + 1)] = {
-                str(location): str(location.item) for location in sorted(sphere)}
+        sphere_index = 1
+        for sphere in collection_spheres:
```

Comment:
Is there a reason you got rid of the enumerate? I'd prefer keeping it and just renaming the variable.

Reply:
I need to skip incrementing the index if a sphere has no locations that can show in spoiler, no need to enumerate it.

Reply:
it would be the same though? you're checking if the sphere has locations and then continuing. you're iterating *over* the enumeration, not *in* it.